### PR TITLE
When clone_with_suffix for level copy the level_concept_difficulty too

### PR DESCRIPTION
**Pull request by dmcavoy**
When we clone_with_suffix on a level (which is what we use when we do a script copy) well will now also copy the level concept difficulty over. This will help with making sure that we don't lose level_concept_difficulty tagging between years of courses. 

I also removed the one-off script we used to do this copy this year since we should no longer need it.

## Links

https://codedotorg.atlassian.net/browse/PLAT-1075

## Testing story

Added a test to make sure the level concept difficulty is copied to the new level

### CODE REVIEW

File: dashboard/app/models/levels/level.rb [hunk 1]
```diff
@@ -733,6 +733,12 @@ def clone_with_suffix(new_suffix, editor_experiment: nil)
     end
 
     level.update!(update_params)
+
+    # Copy the level_concept_difficulty of the parent level to the new level
+    new_lcd = level_concept_difficulty.dup
+    level.level_concept_difficulty = new_lcd
+    level.save! if level.changed?
```
Comment (Hamms) [inline dashboard/app/models/levels/level.rb:740]: I'd recommend we implement this in a way that matches the pattern established by the rest of the method.

```suggestion
    # Copy the level_concept_difficulty of the parent level to the new level
    update_params[:level_concept_difficulty] = level_concept_difficulty.dup
    
    level.update!(update_params)
```

**Review by Hamms — APPROVED** [review]

**Comment by dmcavoy** [comment]
@Hamms I don't think your suggestion works because it looks like assign_attributes only works on properties. The test fails with error. I'm going to put it back to how it was

**Comment by Hamms** [comment]
oh, I believe for properties you need to do something like

```ruby
update_params['properties'] = {level_concept_difficulty: level_concept_difficulty.dup}
```

see https://github.com/code-dot-org/code-dot-org/blob/staging/dashboard/app/models/concerns/serialized_properties.rb

**Comment by Hamms** [comment]
Oh, I think you need to use a string rather than a symbol. See https://github.com/code-dot-org/code-dot-org/blob/545cf4055eaac37b33836c2304e0598f4de7f248/dashboard/app/models/levels/level.rb#L158-L172

**Comment by dmcavoy** [comment]
That looks like it works for something written like this:

```
    level.assign_attributes(
      'level_concept_difficulty' => {'debugging' => 5}
    )
```

Which I think is harder to get to here. So I'm going to merge it as is.

**Pull request merged**